README.md

```diff
@@ -6,7 +6,7 @@
 
 Macros for including transformed files at compile time.
 
-The macros include a file similar to [`include_bytes!`], but transform the data before inclusion (e.g. runs an assembler).
+The macros include a file similar to [`include_bytes!`], but transform the data before inclusion. The transformation might be assembling assembly code to machine code ([`include_nasm_bin!`]) for example.
```

Comment:
_The conversion could be, for example, assembling assembly code into machine code_